src/dsp_tools/commands/excel2xml/excel2xml_lib.py

```diff
@@ -52,22 +52,38 @@ def make_xsd_id_compatible(string: str) -> str:
         BaseError: if the input cannot be transformed to an xsd:ID
 
     Returns:
-        an xsd ID based on the input string
+        an xsd ID based on the input string with a UUID attached.
     """
+    res = xsd_id_remove_illegal_characters(string)
+    # add uuid
+    _uuid = uuid.uuid4()
+    res = f"{res}_{_uuid}"
+    return res
+
+
+def xsd_id_remove_illegal_characters(string: str) -> str:
```

Comment:
IMHO it is not clear enough how the old and the new function relate to each other. I suggest renaming the old into `make_xsd_id_compatible_with_uuid()`, and the new into `make_xsd_id_compatible()`.

The 2 should have similar names, so that people realize that they do similar things.

Reply:
I will not rename the old one because that would be a breaking change.

Reply:
Then we should rename the new one, e.g. into `make_xsd_id_compatible_without_uuid()`